src/Microsoft.Azure.SignalR.Common/ServiceConnections/ClientConnectionScope.cs

```diff
@@ -52,7 +52,7 @@ public void Dispose()
 
         internal static bool IsScopeEstablished => ScopePropertiesAccessor<ClientConnectionScopeProperties>.Current != null;
 
-        internal static IServiceConnection OutboundServiceConnection
+        internal static WeakReference<IServiceConnection> OutboundServiceConnection
```

Comment:
It would look cleaner if we returned `IServiceConnection` here instead of a `WeakReference`